examples/hybrid/package.json

```diff
@@ -12,7 +12,7 @@
   "dependencies": {
     "@react-native-community/cli": "^14.1.1",
     "@react-native-community/cli-platform-android": "^14.1.1",
-    "instabug-reactnative": "^13.4.2",
+    "instabug-reactnative": "file:../../InstabugSDK.tgz",
```

Comment:
I think it needs to be `link:../..` like we have in the `default` example app. Having it `file:../../InstabugSDK.tgz` would require us to build and pack the SDK with every change to reflect in the hybrid app.